client/src/pages/Dashboard/components/utils/makeBlockOnVoiceflow.js

```diff
@@ -0,0 +1,73 @@
+import axios from "axios"
+import uniqid from 'uniqid';
+
+const addBlock = async (diagramID, email, password, intent) => {
+    const token = await login(email, password)
+    const url = `https://api.voiceflow.com/v2/diagrams/${diagramID}/nodes/bulk`
+
+    const id1 = uniqid()
+    const id2 = uniqid()
+    console.log(id1)
```

Comment:
remove the console.log

Reply:
Oops, left that in for testing. Removed!